easybuild/easyblocks/o/openfoam.py

```diff
@@ -169,12 +184,13 @@ def configure_step(self):
         for wmake_rules_file in wmake_rules_files:
             fullpath = os.path.join(self.builddir, self.openfoamdir, wmake_rules_file)
             self.log.debug("Patching compiler variables in %s", fullpath)
-            for line in fileinput.input(fullpath, inplace=1, backup='.orig.eb'):
-                for comp_var, newval in comp_vars.items():
-                    line = re.sub(r"^(%s\s*=\s*).*$" % re.escape(comp_var), r"\1%s" % newval, line)
-                sys.stdout.write(line)
+            regex_subs = []
+            for comp_var, newval in comp_vars.items():
+                regex_subs.append((r"^(%s\s*=\s*).*$" % re.escape(comp_var), r"\1%s" % newval))
+            apply_regex_substitutions(fullpath, regex_subs)
 
         # enable verbose build for debug purposes
+        # starting with openfoam-extend 3.2, PS1 also needs to be set
```

Comment:
so, we should define `$PS1` if it's not set then, to get verbose output when OpenFOAM is built as a job too?

not sure if that would have other fallout, so let's not do it in this PR please, since it's undergoing heavy testing currently to get it in for EB v2.5.0